firewood/src/merkle/node.rs

```diff
@@ -934,6 +934,31 @@ mod tests {
         check_node_encoding(node);
     }
 
+    #[test_case(&[])]
+    #[test_case(&[0x00])]
+    #[test_case(&[0x0F])]
+    #[test_case(&[0x00, 0x00])]
+    #[test_case(&[0x01, 0x02])]
+    #[test_case(&[0x00,0x0F])]
+    #[test_case(&[0x0F,0x0F])]
+    #[test_case(&[0x0F,0x01,0x0F])]
+
+    fn encoded_branch_node_bincode_serialize(path_nibbles: &[u8]) -> Result<(), Error> {
```

Comment:
Any reason not to do this?
```suggestion
    #[test_case(&[0x0F,0x01,0x0F])]
    fn encoded_branch_node_bincode_serialize(path_nibbles: &[u8]) -> Result<(), Error> {
```